public/app/features/alerting/components/AlertingQueryEditor.tsx

```diff
@@ -43,7 +57,7 @@ export class AlertingQueryEditor extends PureComponent<Props, State> {
 
   onNewAlertingQuery = () => {
     const { onChange, value = [] } = this.props;
-    const { defaultDataSource } = this.state;
+    const defaultDataSource = getDatasourceSrv().getInstanceSettings('default');
 
     if (!defaultDataSource) {
       return;
```

Comment:
We have an issue to display an error message if we can't find any default datasource.